<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPropertiesDialog</class>
 <widget class="QDialog" name="UserPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Project Properties</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;User Project Properties&lt;/b&gt;
&lt;p&gt;This dialog is used to show and edit the user specific project properties.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>VCS Status Monitor</string>
     </property>
     <layout class="QHBoxLayout" name="_3">
      <item>
       <widget class="QSpinBox" name="vcsStatusMonitorIntervalSpinBox">
        <property name="toolTip">
         <string>Select the interval in seconds for VCS status updates (0 to disable)</string>
        </property>
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="vcsGroup">
     <property name="title">
      <string>VCS Interface</string>
     </property>
     <layout class="QVBoxLayout" name="_4">
      <item>
       <widget class="QComboBox" name="vcsInterfaceCombo">
        <property name="toolTip">
         <string>Select the vcs interface to be used</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vcsInterfaceDefaultCheckBox">
        <property name="toolTip">
         <string>Select to make the interface selection the default for the project</string>
        </property>
        <property name="text">
         <string>Make interface selection the default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vcsStatusMonitorIntervalSpinBox</tabstop>
  <tabstop>vcsInterfaceCombo</tabstop>
  <tabstop>vcsInterfaceDefaultCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
